cosmos-server/src/main/scala/com/mesosphere/cosmos/converter/Response.scala

```
@@ -19,26 +21,36 @@ object Response {
   }
 
   def packageDefinitionToDescribeResponse(
-      packageDefinition: internal.model.PackageDefinition)
-    : rpc.v1.model.DescribeResponse = {
-    // TODO(version): We are throwing. Not sure this is the right thing
-    packageDefinition
-      .as[Try[universe.v3.model.V2Package]]
-      .map { v2Package =>
-        rpc.v1.model.DescribeResponse(
-            `package` = v2Package.as[universe.v2.model.PackageDetails],
-            marathonMustache =
-              new String(universe.common.ByteBuffers
-                           .getBytes(v2Package.marathon.v2AppMustacheTemplate),
-                         Charsets.Utf8),
-            command = v2Package.command.as[Option[universe.v2.model.Command]],
-            config = v2Package.config,
-            resource =
-              v2Package.resource.as[Option[universe.v2.model.Resource]]
-        )
-      }
-      .get
+      packageDefinition: internal.model.PackageDefinition): Try[rpc.v1.model.DescribeResponse] = {
```

It's the marathon template: optional in `internal.model.PackageDefinition`, but required in `rpc.v1.model.DescribeResponse`. I'll update the code to throw a `ConversionFailure` instead of returning a `Try`. In the long term though it would be a good idea to use a functional error handling mechanism to accumulate all conversion errors.